ror-demo-cluster/conf/es/readonlyrest.yml

```
@@ -18,46 +18,56 @@ readonlyrest:
-        - id: "EndUsers"
-          name: "End Users"
-        - id: "BusinessUsers"
-          name: "Business Users"
-
-    - username: user2
-      auth_key: user2:test
-      groups:
-        - id: "EndUsers"
-          name: "End Users"
+
+  - username: admin
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)